docs/core/tools/sdk-errors/netsdk1045.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "NETSDK1045: The current .NET SDK does not support 'newer version' as a target."
+description: Learn about .NET SDK error NETSDK1045, which occurs when the build tools can't find the requested version of the .NET SDK.
+ms.topic: error-reference
+ms.date: 02/12/2021
+f1_keywords:
+- NETSDK1045
+---
+# NETSDK1045: The current .NET SDK does not support 'newer version' as a target. Either target 'older version' or lower, or use a .NET SDK version that supports 'newer version'.
```

Comment:
Per [SEO guidance](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-basics?branch=master), H1 should be limited to 100 characters.

```suggestion
# NETSDK1045: The current .NET SDK does not support 'newer version' as a target.
```